docs/reST/ref/cursors.rst

```diff
@@ -174,6 +176,12 @@ The following strings can be converted into cursor bitmaps with
       pygame.SYSTEM_CURSOR_NO          slashed circle or crossbones
       pygame.SYSTEM_CURSOR_HAND        hand
 
+   **Creating a cursor without passing arguments**
+   
+   In addition to the cursor constants available and described above,
+   you can also call ``pygame.cursors.Cursor()``, and your cursor is ready.
+   Doing this call actually creates a system cursor using the default native image.
```

Comment:
maybe this should just say it's the same as pygame.SYSTEM_CURSOR_ARROW, because that's what it actually does.

Reply:
Done